src/App.tsx

```diff
@@ -2,18 +2,152 @@ import React from 'react';
 import './App.css';
 
 import users from './api/users';
+import todos from './api/todos';
+import { State } from './types/StateForApp';
+import { TodoList } from './components/TodoList';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+class App extends React.Component<{}, State> {
+  state = {
+    todosList: [...todos],
+    title: '',
+    selectedUser: '',
+    todo: '',
+    titleIsValid: true,
+    userIsSelected: true,
+  };
+
+  render() {
+    const {
+      todosList, title, selectedUser, titleIsValid, userIsSelected,
+    } = this.state;
+
+    const todosForRender = todosList.map((todoApp) => ({
```

Comment:
```suggestion
    const todosForRender = todosList.map((todo) => ({
```
Один елемент тудуліста - туду. Тому, варто використвоувати більш підходящі назви